site/docs/tasks/multicluster.md

```diff
@@ -106,3 +106,80 @@ For the remote cluster the coordinator will be disabled:
 coordinator:
     enabled: false
 ```
+
+
+## Configure Vault for multi-cluster deployment
+
+The Fybrik uses [HashiCorp Vault](https://www.vaultproject.io/) to provide running Fybrik modules in the clusters with the dataset credentials when accessing data. This is done using [Vault plugin system](https://www.vaultproject.io/docs/internals/plugins) as described in [here](../concepts/vault_plugins.md).
+
+
+This section describe the steps to enable the modules to authenticate to Vault in order for them to retrieve the dataset credentials.
```

Comment:
This section describe**s** steps to enable the module's Vault authentication, in order to obtain database credentials.

Reply:
Maybe "... to authenticate with Vault..."